source/reference/object.rst

```diff
@@ -73,17 +73,22 @@ conventionally referred to as a "property".
 .. index::
    single: object; properties
    single: properties
-   single: additionalProperties
 
-.. _additionalProperties:
+.. _properties:
 
 Properties
 ''''''''''
 
 The properties (key-value pairs) on an object are defined using the
 ``properties`` keyword.  The value of ``properties`` is an object,
-where each key is the name of a property and each value is a JSON
-schema used to validate that property.
+where each key is the name of a property and each value is a schema
```

Comment:
I removed the "JSON" for consistency. I've tried to stick with the general conventions that were originally used. Almost always, when referring to a schema, "schema" or "subschema" is used rather than "JSON Schema". "JSON Schema" is used when referring to the specification.

Reply:
Fair. I've settled on using "Schema" upper case.